packages/fxa-settings/src/components/Settings/index.tsx

```diff
@@ -67,6 +73,10 @@ export const Settings = ({
 
   const { loading, error } = useInitialSettingsState();
 
+  useEffect(() => {
+    !loading && GleanMetrics.pageLoad(location.pathname);
```

Comment:
@chenba Moving the page load event from `App` level to here appears to work to pull in the missing metrics data (e.g., flow id) for the Settings pages, but attempting a similar approach for the non-Settings pages caused a ripple effect of issues for tests and storybooks. Maybe there's a better approach for `FXA-10323`